InteractionBase/src/commands/Command.h

```diff
@@ -52,6 +53,16 @@ class INTERACTIONBASE_API Command
 				const QString& textSoFar, const std::unique_ptr<Visualization::Cursor>& cursor) = 0;
 		virtual QList<CommandHelp*> extendedHelp(Visualization::Item* source, Visualization::Item* target,
 				const std::unique_ptr<Visualization::Cursor>& cursor, const QString& commandForm = QString());
+        bool canBeUsedInMenu() const;
+        const QString name() const;
+
+    private:
+        bool canBeUsedInMenu_;
```

Comment:
Primitive types such as int, bool, float, double and all pointer types should always be initialized with some value. (This is a safety rule in Envision, not a C++ thing). This means that in the class declaration in the .h file you should at least default initialize them:
```bool canBeUsedInMenu_{}; // note the extra {}```
The {} syntax works for all types (that have a default constructor)